data-prepper-plugins/otel-metrics-raw-processor/README.md

```diff
@@ -101,6 +104,12 @@ a scale parameter, offset and list of bucket counts:
     "positiveOffset : 1
 ```
 
+The `exponential_histogram_max_allowed_scale` parameter defines the maximum allowed scale for the exponential histogram. Increasing this parameter will increase potential
+memory consumption. See [the spec](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto) for more information on exponential histograms and their computational complexity.
+All exponential histograms that have a scale that is above 10 (or the configured parameter) will be discarded and logged with error level.
```

Comment:
> that is above 10 (or the configured parameter)

I think this might be somewhat clearer.

> that is above the configured parameter (by default, 10).